server/datastore/mysql/migrations/tables/20220708095046_AddUniqconstraintSoftwareIDOnSoftwareCVE.go

```diff
@@ -0,0 +1,149 @@
+package tables
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/fleetdm/fleet/v4/server"
+	"github.com/pkg/errors"
+)
+
+var logger *Logger
```

Comment:
Could you please elaborate a little bit more on this? Not sure what you mean by 'declaring' and 'initializing' in `log.go`

Reply:
move the following to `log.go`, since it could be used in more than one migration.

```
var logger = NewLogger()
```

Also noticed a couple `fmt.Println` statements in older migrations that could be changed to use the same logger eg `server/datastore/mysql/migrations/tables/20220704101843_AddSoftwareIDInSoftwareCVE.go`